ge/org.osate.ge/src/org/osate/ge/internal/ui/handlers/AlignmentUtil.java

```diff
@@ -0,0 +1,357 @@
+package org.osate.ge.internal.ui.handlers;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.osate.ge.graphics.Point;
+import org.osate.ge.internal.diagram.runtime.DiagramElement;
+import org.osate.ge.internal.diagram.runtime.DiagramModification;
+import org.osate.ge.internal.diagram.runtime.Dimension;
+import org.osate.ge.internal.diagram.runtime.DockArea;
+import org.osate.ge.internal.query.Queryable;
+
+
+class AlignmentUtil {
```

Comment:
Rename to AlignmentHelper for consistency with other code. If a utility class is intended to be instantiated to be used, we name it with a Helper suffix. If it is just static methods, name it with the "Util" suffix.